beetsplug/convert.py

```diff
@@ -183,8 +185,8 @@ def commands(self):
 
     def auto_convert(self, config, task):
         if self.config['auto']:
-            for item in task.imported_items():
-                self.convert_on_import(config.lib, item)
+            items = zip(repeat(config.lib), list(task.imported_items()))
+            par_map(self.convert_on_import, items)
```

Comment:
Maybe make this `par_map(lambda item: self.convert_on_import(config.lib, item), task.imported_items())` with the old function signature of `convert_on_import`. In my opinion, that would be a lot clearer and should do precisely the same thing, right? I don't think there's any need to make a list out of `task.imported_items()`.

Reply:
done